docs/site/Controllers.md

```diff
@@ -10,83 +10,109 @@ summary:
 
 ## Overview
 
-A `Controller` is a class that implements operations defined by application's REST API. It implements an application's business logic and acts as a bridge between the HTTP/REST API and domain/database models.
-A `Controller` operates only on processed input and abstractions of backend services / databases.
+A `Controller` is a class that implements operations defined by application's
+REST API. It implements an application's business logic and acts as a bridge
+between the HTTP/REST API and domain/database models. A `Controller` operates
+only on processed input and abstractions of backend services / databases.
```

Comment:
Controllers can be used for other API forms beyond REST. We add decorations to a controller class and its members to describe how to map JavaScript methods to API operations.

Reply:
We can say this page only covers REST APIs for controllers.